content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -61,6 +61,20 @@ among the comma-delimited, ordered list of values for the `--enable-admission-pl
 the API server component. For more information on API server command-line flags,
 check [kube-apiserver](/docs/admin/kube-apiserver/) documentation.
 
+Claims can also be created without any StorageClass while no default exists. In that case
+their StorageClass will remain unset until a default becomes available, then the claim
+will get the new default assigned by `RetroactiveDefaultStorageClass` [feature](/docs/concepts/storage/persistent-volumes/#retroactive-storageclass-assignment).
+
+### Retroactive StorageClass assignment
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+A PVC can be created without specifying any StorageClass. This can happen either if there is no default StorageClass set by the administrator or [`DefaultStorageClass` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass) is disabled. PVs with `StorageClassName` set to `""` will continue to bind to such PVCs until default StorageClass is created.
+
+When default StorageClass becomes available any existing PVCs without `StorageClassName` will get this value updated retroactively to the default StorageClass name. In order to keep binding to PVs with `StorageClassName` set to `""` (while default StorageClass is present) users need to set PVC `StorageClassName` to `""`.
```

Comment:
When a default StorageClass becomes available, the control plane identifies any existing PVCs with an empty`storageClassName`. For the PVCs that have an empty value `storageClassName`, the control plane then updats those PVCs to set `sotrageClassName` to match the new default StorageClass.

In order to keep binding to PVs with `storageClassName` set to `""` (while default StorageClass is present), you need to set the `storageClassName` of the associated PVC to `""`.
```

Reply:
Same as above, the word empty needs to be used with caution. Because empty string (`""`) is exactly and the only one case when PVC will **not** get updated.

Reply:
Looks good, I've updated it to match the form suggested and added more details.